dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -202,7 +202,7 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 			"bin/alluxio-stop.sh",
 			"bin/alluxio-workers.sh",
 			"bin/launch-process",
-			fmt.Sprintf("client/alluxio-%v-client.jar", version),
+			fmt.Sprintf("client/build/alluxio-%v-hdfs2-client.jar", version),
```

Comment:
is the default compiling with hadoop 2? I thought it's 3?

Reply:
also i guess hadoop2 is better as a name than hdfs2 because it is more generic. My gut feeling is people more often say "hadoop 2 or 3" than "hdfs 2 or 3".

Reply:
> is the default compiling with hadoop 2? I thought it's 3?

so actually i'm confused about that too. i thought we were using 3 but the blocked PR needs a hadoop 3 shaded jar?